Desktop/modules/dynamicmodules.cpp

```diff
@@ -57,7 +58,7 @@ DynamicModules::DynamicModules(QObject *parent) : QObject(parent)
 
 DynamicModules::~DynamicModules()
 {
-	for(auto dynamic : _modules)
+	for(const auto & dynamic : _modules)
```

Comment:
A DynamicModule is a QObject with parent this DynamicModules QObject. So why do you have to delete it. It's even dangerous.

Reply:
That is a good point!

Reply:
Remnant from before, when they werent qobjects yet!
I just made the code look a bit better but didnt think about it.